tests/pages/ShowIdentityPage.test.tsx

```diff
@@ -185,32 +167,26 @@ describe("ShowIdentityPage", () => {
 
   test("it should not display the primary tag if the identity is not primary", () => {
     const component = mount(
-      <ThemeProvider theme={lightTheme}>
-        <GlobalStyle />
-        <Layout>
-          <ShowIdentityPage identity={nonPrimaryIdentity} />
-        </Layout>
-      </ThemeProvider>,
+      <AccountApp>
+        <ShowIdentityPage identity={nonPrimaryIdentity} />
+      </AccountApp>,
     );
 
     const primaryBadge = component.contains(<PrimaryBadge />);
-    const makeThisPrimaryButton = component.contains(
-      <Button variant={ButtonVariant.SECONDARY}>
-        Make this my primary email
-      </Button>,
-    );
+    const makeThisPrimaryButton = component
+      .find(Button)
```

Comment:
I guess that `component.find()` returns **all** the `Button` it finds, right?

Reply:
Yes, I had to implement this way to find the button since it now has an `onClick` property that would be tricky to mock in this test file.
However, I'm checking on line 182 that the button I'm testing is the one I'm expecting